File: Fonte/Back/ListaTarefa/ListaTarefa.Domain/Pagination/Paging/TarefaPageMessage.cs
using ListaTarefa.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ListaTarefa.Domain.Pagination.Paging
{
    public class TarefaPageMessage
    {
        public int PageNumber { get; set; }
        public int PageSize { get; set; }
        public int TotalCount { get; set; }
        public int TotalPages { get; set; }
        public bool HasPrevious { get; set; }
        public bool HasNext { get; set; }
        public List<Tarefa> Items { get; set; }
    }
}

File: Fonte/Front/ListaTarefa.Front/src/app/core/model/_pageMessage/tarefaPageMessage.ts
import { Tarefa } from "../tarefa";

export class TarefaPageMessage {
    pageNumber: number;
    pageSize: number;
    totalCount: number;
    totalPages: number;
    hasPrevious: boolean;
    hasNext: boolean;
    items: Tarefa[];
  
    constructor(pageNumber: number, pageSize: number, totalCount: number, totalPages: number, hasPrevious: boolean, hasNext: boolean, items: Tarefa[]) {
      this.pageNumber = pageNumber;
      this.pageSize = pageSize;
      this.totalCount = totalCount;
      this.totalPages = totalPages;
      this.hasPrevious = hasPrevious;
      this.hasNext = hasNext;
      this.items = items;
    }
  }